logic.js

```diff
@@ -0,0 +1,56 @@
+
+
+const ECommerce = function()
+{
+
+    let itemsInCart = 2;
+
+    const cart = [
+        {name: 'mac', price: 9000, img:"images/mac.jpeg"},
+        {name: 'iphone', price: 5000, img:"images/iphone.png"}
+    ]
+
+    const products = [
+        {name: 'mac', price: 9000, img: "images/mac.jpeg"},
+        {name: 'iphone', price: 5000, img:"images/iphone.png"},
+        {name: 'airpods', price: 1000, img:"images/airpods.jpg"},
+        {name: 'ipad', price: 3000, img:"images/ipad.jpg"},
+        {name: 'appleWatch', price: 2000, img:"images/appleWatch.jpg"}
+    ]
+
+    const getProducts = () => products
+    const getCart = () => cart
+
+    const addToCart = function(product){
+        itemsInCart++;
+        cart.push(product)
+    } 
+    
+
+    const deleteFromCart = function(product){
+        itemsInCart--;
+        const Pname = product.name;
```

Comment:
Shorter veriables are not always better. You need to make sure your code is super easy for someone to read. Making a variable a bit longer can help with this. 

So instead of `Pname` make it `productName`

also make sure you camelCase in JS so it should have been `pName`